mocks/tests/test_mocks.py

```diff
@@ -34,8 +34,15 @@ def pixelated_psf_data(huntsman_sbig_dark_imager):
     return psf_data
 
 
-def test_prepare_mocks():
-    mocks_config, data = mocks.prepare_mocks()
+def test_load_configuration():
+    config = mocks.load_configuration()
+    assert config['data_path'] == 'sim_data/cl19.fits'
+    assert config['galaxy_distance'] == 10.
+    assert type(config) == dict
```

Comment:
Generally speaking `isinstance(object, class)` is preferred for testing the type of an object, not `type(object) == class`. Several reasons for that, the main one being that `isinstance` allows for subclasses.